controllers/tags.js

```diff
@@ -63,6 +75,27 @@ exports.relatedToMyTags = async function (req, res, next) {
   }
 };
 
+exports.relatedToTags = async function (req, res, next) {
+
+  const tagsArray = req.query.filter.relatedToTags.split(',');
+
+  try{
```

Comment:
did it pass `gulp eslint`? seems like formatting error. (space needs to be in `try {`)